media/css/firefox/tiles.less

```diff
@@ -4,10 +4,136 @@
 
 @import "../sandstone/lib.less";
 
+#outer-wrapper {
+    overflow: hidden;
+}
+
 #wrapper {
     margin-top: 50px;
 }
 
-.sidebar .sidebar-inset img {
-    border: none;
+.sidebar {
+    float: left;
+    margin: -200px 10px 0 10px;
+
+    .sidebar-inset {
+        background-color: transparent;
+        box-shadow: none;
+        border: none;
+        padding: 0;
+
+        img {
+            border: none;
+            max-width: none;
+            margin-bottom: @gridGutterWidth;
+        }
+
+        img.tab {
+            width: 200%;
+        }
+
+        img.tab-preferences {
+            width: 100%;
+        }
+    }
+}
+
+.main-column {
+    width: 580px;
+
+    .intro {
+        font-size: @baseLine;
+        font-size: 1.2rem;
+        font-style: italic;
+        font-weight: 100;
+        padding-bottom: 2em;
+    }
+
+
+    section {
+        padding-top: 2.5em;
+        border-top: solid 3px @borderColor;
+
+        h4 {
+            font-weight: bold;
+            font-style: italic;
```

Comment:
@schalkneethling Great! I didn't even realize this! Will add font-family: "Open Sans"; Yes I agree italic is lesser of two evils. I don't think it's necessary to import required font as it looks great now that it's bold.